crates/router/src/core/payments.rs

```diff
@@ -455,6 +464,64 @@ where
                     } else {
                         None
                     };
+                    let filtered_nt_supported_connectors =
+                        get_filtered_nt_supported_connectors(&state, [connector.clone()].to_vec());
+
+                    let is_nt_supported_connector_available =
+                        filtered_nt_supported_connectors.first().is_some();
```

Comment:
do no filter out. check only if the connector decided by routing is in the network tokenization supported connectors. this is because routing should always be prioritized over any feature. please remove this.